tests/test_query.py

```diff
@@ -114,6 +113,7 @@ def test_get_valid_age_single_bound(
     assert response.json() != []
 
 
+@pytest.mark.parametrize("mock_get", [None], indirect=True)
```

Comment:
phew. That's cool, but I see what you mean, bit tricky to "just read". I guess what this tells me is that you tell `mock_get` to have a `None` return here?

Reply:
I wonder if we haven't gone a little over-complex here. Pytest seems to have much simpler ways to pass data to fixtures from tests: https://docs.pytest.org/en/6.2.x/fixture.html#using-markers-to-pass-data-to-fixtures . I believe that would be a lot more readable. Wdyt?

Reply:
After looking into this, sticking with the original indirect parametrization approach seems to be more intuitive in terms of syntax and from a test-writing perspective, while still helping us avoid re-mocking get functions inside individual tests. So proceeding with the original changes!